src/main/php/Handler/LokiHandler.php

```diff
@@ -90,7 +94,7 @@ private function sendPacket(array $packet): void
         }
 
         if (false !== $this->connection) {
-            $curlOptions = [
+            $curlOptions = $this->customCurlOptions + [
```

Comment:
```suggestion
            $curlOptions = array_merge($this->customCurlOptions, [
```

Also there's something missing in your change : the symfony service configuration example is missing the parts related to your changes.

(BTW I noticed it is also missing the auth parts, could you use the opportunity to add them as well)

Reply:
`$curlOptions` doesn't look like it, but it has numerical keys (The `CURLOPT_*` constants contain integer values).

`array_merge()` re-indexes numeric arrays, so the wrong and/or unknown curl options are set. Using `+` instead of `array_merge()` is correct here.


I'll add the custom curl options and the auth to the Symfony example as well.

Reply:
Yes my bad. 
Worst part,  I went and checked the constants but didn't register that problem (even though I am aware of it) -_-
I don't really like using the + operator for arrays, array_replace should do the trick as it works fine with numerical keys.

Reply:
Applied `array_replace`, that looks more readable indeed.

Concerning the custom curl options in the Symphony sample: That probably won't work, because the `CURLOPT_*` PHP constants don't get replaced by their values in YAML. Or does Symphony some magic on that YAML?

Reply:
For the symfony version, there is a specific syntax for YAML ; `!const {constValue}` (see https://symfony.com/doc/current/configuration.html#configuration-parameters)

Reply:
I've added the parameters to the Symfony sample with the `!php/const GLOBAL_CONSTANT` syntax.